integreat_cms/cms/models/contact/contact.py

```diff
@@ -127,6 +127,16 @@ def search(cls, region: Region, query: str) -> QuerySet:
 
     @classmethod
     def search_for_query(cls, region: Region, query: str) -> QuerySet:
+        """
+        Searches for all contacts in the specified region whose name or location information
+        matches the search query. The match is case-insensitive and supports partial text matching.
+        :param region: The region to filter contacts by.
+        :type region: Region
```

Comment:
```suggestion
```
Since we already have the type in the method head, we usually don't also put the type in our class comments. This is not a big issue to me, but at least wanted to let you know about it :)